R/alanwood.R

```diff
@@ -8,9 +8,8 @@
 #' @param  query character; search string
 #' @param type character; type of input ('cas' or 'commonname')
 #' @param verbose logical; print message during processing to console?
-#' @param idx data.frame; index to use. If NULL (default) the internal index
-#' \code{\link{aw_idx}} is used. To rebuild the index use
-#' \code{\link{build_aw_idx}}.
+#' @param force_build logical; force building a new index? See
+#' \code{build_aw_idx()} for more details.
```

Comment:
add a link to this help file.
```suggestion
#' \code{\link{build_aw_idx()}} for more details.
```